test/gui/shared/scripts/pageObjects/PublicLinkDialog.py

```diff
@@ -95,7 +95,7 @@ class PublicLinkDialog:
     defaultExpiryDate = ""
```

Comment:
```suggestion
    DEFAULT_EXPIRY_DATE = ""
```

Reply:
hmm, the linter didn;t catch the class members

Reply:
this rule was missing `class-attribute-naming-style`